src/atoms/TextInput/TextInput.native.js

```diff
@@ -14,18 +14,19 @@ import automation from '../../_helpers/automation-attributes';
 import View from '../View';
 import isEmpty from '../../_helpers/isEmpty';
 import Size from '../Size';
+import Space from '../Space';
 
 const IS_ANDROID = Platform.OS === 'android';
 
 const styles = {
   textInput: {
     padding({ variant, hasLeftIcon, hasPrefix, hasText }) {
-      const paddingTop = IS_ANDROID ? '8px' : '0px';
-      const paddingRight = '0px';
-      const paddingBottom = IS_ANDROID ? '0px' : hasText ? '4px' : '2px';
-      const paddingLeft = variant === 'outlined' || hasLeftIcon || hasPrefix ? '0px' : '8px';
+      const paddingTop = IS_ANDROID ? (variant === 'filled' ? 0 : 1) : 0;
```

Comment:
Just avoiding nested ternary operations.
```
let [paddingTop, paddingRight,paddingBottom, paddingLeft ] = [0,0,0,0];

if(IS_ANDROID){
  paddingTop = variant === 'filled' ? 0 : 1;
}else{
  paddingBottom = hasText ? 0.5 : 0.25;
}

paddingLeft = variant === 'outlined' || hasLeftIcon || hasPrefix ? 0 : 1;

return [paddingTop, paddingRight,paddingBottom, paddingLeft];
```

Reply:
Isn't this rule enabled on eslint? [`no-nested-ternary`](https://eslint.org/docs/rules/no-nested-ternary)

Reply:
I don't think so. It's coming from universe

Reply:
[It's off currently.](https://github.com/kentcdodds/eslint-config-kentcdodds/blob/8e5720c101c895856f615b7de7b549114b1b02fc/stylistic.js#L49)
@kamleshchandnani